File: Selenium/Activities/Activity_4.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_4 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/target-practice");
        System.out.println("The title of Webpage is:  " + driver.getTitle());
        System.out.println("the third header element text is: " + driver.findElement(By.id("third-header")).getText());
        WebElement element = driver.findElement(By.xpath("//h5[@class='ui green header']"));
        String cssValue = element.getCssValue("color");
        System.out.println("The colour of fifth element is: " + cssValue);

        WebElement ele2 = driver.findElement(By.xpath("//button[text()='Violet']"));
        String attr = ele2.getAttribute("class");
        System.out.println("The Class name of Voilet button is: " + attr);
        System.out.println("the Grey button text is: " + driver.findElement(By.xpath("//button[text()='Grey']")).getText());

        driver.quit();
    }
}

File: Selenium/Activities/Activity_4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("The Title of WebPage is:", driver.title)
    third_ele = driver.find_element(By.XPATH, "//h3[text()='Third header']")
    print("The text of third element is", third_ele.text)
    element = driver.find_element(By.XPATH, "//h5[@class='ui green header']")
    print("The color of fifth header is ", element.value_of_css_property("color"))
    ele1 = driver.find_element(By.CLASS_NAME, "violet ")
    print("the class name of voilet button is: ", ele1.get_attribute("class"))
    print("the text of grey button is ", driver.find_element(By.CLASS_NAME, "grey").text)
    driver.quit()